.github/CODEOWNERS

```
@@ -0,0 +1 @@
+*.png @jni
```

I invite you to make suggestions if you wish to add yourself to a part of the codebase, like so:

```suggestion
*.png @jni
napari/plugins/ @tlambert03 @nclack 
```

---

looking here, more files should be assigned to jni. 
On the other hand, each part should be at least two people not to have a block during someone's holidays.

https://github.com/napari/napari/blob/297dfc3dfc48963181411d64d06199857076ca4c/.gitattributes#L2-L5

---

Sure, in the end I expect teams to be assigned, and not single contributors.

---

```suggestion
*.png @napari/core-devs 
*.jpg @napari/core-devs
*.gif @napari/core-devs
*.webm @napari/core-devs
```